core/agent.py

```
@@ -47,111 +110,95 @@ def run(
+            self._stop_condition = stop_condition
+
+            # 初始化对话
+            self.conversation = [
+                {"role": "system", "content": self._get_system_prompt()},
+                {"role": "user", "content": task},
+            ]
+
+            print("🤖 NanoAgent - 极简 Agent 框架")
+            print(f"📋 任务: {task}")
+
+            # 主循环 - 无限制，直到条件满足
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)